campaignion_tracking/package/src/drupal-common.js

```diff
@@ -0,0 +1,84 @@
+import * as tm from './tracker-manager'
+import * as listener from './listener'
+
+/**
+ * Mapping of codes to eventNames.
+ *
+ * This also is a whitelist of which events should *also* be recognized as
+ * codes.
+ */
+export const trackingCodes = {
+  ds: 'donationSuccess'
+}
+
+/**
+ * Check if we want debugging output.
+ *
+ * Parse the value as int, so we can disable debugging by setting to "0".
+ * `sessionStorage` only stores strings.
+ */
+// eslint-disable-next-line no-unneeded-ternary
+var debug = parseInt(sessionStorage.getItem('campaignion_debug')) ? true : false
+
+let printDebug = (...args) => {
+  if (debug) {
+    console.debug('[campaignion_tracking]', '(drupal)', ...args)
+  }
+}
+
+// some out-of-the-box prefixes
+// t: tracking, w: webform, d: donation
+export const codePrefixes = ['t', 'w', 'd']
+
+// common tracker manager, listener, gtm
+export const tracker = new tm.TrackerManager(debug)
+export const fragmentListener = new listener.FragmentListener(tracker, codePrefixes, debug)
+fragmentListener.setup()
+
+/**
+ * Subscribe to messages of the `code:gtm` tracking channel.
```

Comment:
The comment seems to be outdated (still referring to `gtm`).

Reply:
fixed